src/exchange/domain/add/domain-add.controller.js

```diff
@@ -42,20 +41,18 @@ angular
             this.ovhDomain = this.OVH_DOMAIN;
             this.nonOvhDomain = this.NON_OVH_DOMAIN;
 
-            this.services
-                .ovhUserPref
-                .getValue("EXCHANGE_DOMAIN_ADD_AUTO_DISPLAY")
-                .then((res) => {
-                    this.autoDisplay = res.autoDisplay;
-                });
-
             $scope.loadDomainData = () => this.loadDomainData();
             $scope.addDomain = () => this.addDomain();
             $scope.isNonOvhDomainValid = () => this.isNonOvhDomainValid();
             $scope.checkDomain = () => this.checkDomain();
             $scope.isStep2Valid = () => this.isStep2Valid();
             $scope.checkDomainType = () => this.checkDomainType();
             $scope.isStep3Valid = () => this.isStep3Valid();
+
+            User.getUser()
+                .then((currentUser) => {
+                    this.canOpenWizard = currentUser.ovhSubsidiary !== "CA";
+                });
```

Comment:
Sorry Canadians... :cry: 
More seriously, we could make a backend for enabling/disabling features instead of putting this logic in the frontend.

Reply:
It's actually a front-end issue, not a back-end one: the CA web manager is older than the current Web manager

Reply:
Oh I see, my bad.
The idea of a database to enable/disable features still stands though.